plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -1549,12 +1549,12 @@ function woocommerce_catalog_ordering() {
 		$catalog_orderby_options = apply_filters(
 			'woocommerce_catalog_orderby',
 			array(
-				'menu_order' => __( 'Default sorting', 'woocommerce' ),
-				'popularity' => __( 'Sort by popularity', 'woocommerce' ),
-				'rating'     => __( 'Sort by average rating', 'woocommerce' ),
-				'date'       => __( 'Sort by latest', 'woocommerce' ),
-				'price'      => __( 'Sort by price: low to high', 'woocommerce' ),
-				'price-desc' => __( 'Sort by price: high to low', 'woocommerce' ),
+				'menu_order' => __( 'Default', 'woocommerce' ),
+				'popularity' => __( 'Popularity', 'woocommerce' ),
+				'rating'     => __( 'Average rating', 'woocommerce' ),
+				'date'       => __( 'Latest', 'woocommerce' ),
+				'price'      => __( 'Price: low to high', 'woocommerce' ),
+				'price-desc' => __( 'Price: high to low', 'woocommerce' ),
```

Comment:
@dkotter 
Since the "Sort by" text went to the visual label,
![image](https://github.com/user-attachments/assets/4465ddb5-ca8c-4a7f-806b-e11fad1fff33)
it doesn't make sense to repeat it in the option values. However, the array with the options passes through the `woocommerce_catalog_orderby` filter, and plugins and themes might be expecting the values as they were.

Do you have any insights on how to handle this change?

Thanks!

Reply:
So not sure there's a great way for us to handle this. We could in theory modify the code here where we run the filter and then remove any `Sort by` text in those labels after, but doesn't seem ideal for us to force that on others.

I think the correct approach here is to ensure this is properly flagged in release notes / release posts so 3rd parties can make changes if desired. While it's not ideal to show `Sort by` twice, it doesn't break anything so I think as long as we make it clear this change is coming, that's the most we can do

Reply:
Thanks for sharing your thoughts, @dkotter

Reply:
Good question! This seems like a breaking change to me.
- Third party themes and plugins might be relying on these exact strings.
- Documentation might become outdated.
- All existing translations would need updates.

I agree with @dkotter. We will need to atleast publish a Dev advisory for this change.